packages/server/tests/helpers/utils.ts

```diff
@@ -43,4 +47,8 @@ export class Utils {
         return BigNumber.from(hex1).sub(BigNumber.from(hex2));
     };
 
+    static tinyBarsToWeibars = (value) => {
+        return Utils.numberTo0x(Number.parseInt(value) * 10_000_000_000);
```

Comment:
nit: make `10_000_000_000` const

Reply:
Lets use directly the util method from ethers:
```TypeScript
return ethers.utils.hexValue(ethers.utils.parseUnits(value, 10));
```